app/scripts/views/base.js

```diff
@@ -87,10 +87,14 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
     constructor: function (options) {
       options = options || {};
 
+      if (! options.translator) {
+        throw new Error('Translator is required');
+      }
```

Comment:
this is here in the WIP to show when Translator is not passed into a view. 
We probably need some other way to initialize the Translator, rather than passing it through...